apphelpers/rest/hug.py

```diff
@@ -3,11 +3,15 @@
 
 import hug
 from hug.decorators import wraps
-from honeybadger import Honeybadger
 
 from apphelpers.db.peewee import dbtransaction
 from apphelpers.errors import InvalidSessionError
 from apphelpers.sessions import SessionDBHandler
+from converge import settings
+
+if settings.HONEYBADGER_API_KEY:
```

Comment:
Please use `settings.get`

Reply:
Any specific reason to use this ?
`if settings.WHATEVER:` style is used everywhere in `reactor` code (at least)

Reply:
get() makes it safer. If the app don't use honeybadger, API key won't be there.